CHANGELOG.md

```diff
@@ -5,6 +5,193 @@ All notable changes to this project will be documented in this file.
+- [#4492](https://github.com/oscar-system/Oscar.jl/pull/4492) Add links to commutative algebra tutorials
+- [#4488](https://github.com/oscar-system/Oscar.jl/pull/4488) Overhaul preprocessing for `radical`, `primary_decomposition`, and friends over number fields
+- [#4468](https://github.com/oscar-system/Oscar.jl/pull/4468) Graduate `present_finite_extension_ring` from experimental to supported (and fix a bug in it)
+- [#4456](https://github.com/oscar-system/Oscar.jl/pull/4456) Generalize `monomials_of_degree` to allow graded rings with grading group Z
+- [#4400](https://github.com/oscar-system/Oscar.jl/pull/4400) Speed up evaluation of maps of `MPolyRing`s which take variables to variables 
+- [#4379](https://github.com/oscar-system/Oscar.jl/pull/4379) Fix bug in comparison function in module orderings
+- [#4346](https://github.com/oscar-system/Oscar.jl/pull/4346) Fix `characteristic` method for localized rings
+- [#4235](https://github.com/oscar-system/Oscar.jl/pull/4235) Add `monomial_basis` for `MPolyQuoLocRing`
+
+### F-Theory Tools
+
+- [#4636](https://github.com/oscar-system/Oscar.jl/pull/4636) Add attribute for more detailed info on tunable sections
+- [#4597](https://github.com/oscar-system/Oscar.jl/pull/4597) Update `tune` function and related definitions/functions
+- [#4562](https://github.com/oscar-system/Oscar.jl/pull/4562) Align names of properties and attributes among families of and individual G4-fluxes
+- [#4503](https://github.com/oscar-system/Oscar.jl/pull/4503) Fix and extend strict_transform
+- [#4500](https://github.com/oscar-system/Oscar.jl/pull/4500) Extend support for G4-flux families and individual G4-fluxes
+- [#4491](https://github.com/oscar-system/Oscar.jl/pull/4491) Save updated intersection numbers after costly computation
+- [#4487](https://github.com/oscar-system/Oscar.jl/pull/4487) Compute D3-tadpole for family of G4-fluxes
+- [#4466](https://github.com/oscar-system/Oscar.jl/pull/4466) Introduce families of G4-fluxes
+- [#4446](https://github.com/oscar-system/Oscar.jl/pull/4446) Add support for all vertical, well-quantized G4s that do not break the non-abelian gauge group
+- [#4423](https://github.com/oscar-system/Oscar.jl/pull/4423) Use Zenodo data as artifact
+- [#4422](https://github.com/oscar-system/Oscar.jl/pull/4422) Print known properties of G4-fluxes
+- [#4343](https://github.com/oscar-system/Oscar.jl/pull/4343) Extend support for zero section and zero section class
+- [#4286](https://github.com/oscar-system/Oscar.jl/pull/4286) Support computation of all well-quantized and vertical G4-fluxes
+- [#4268](https://github.com/oscar-system/Oscar.jl/pull/4268) Implement ambient space models for G4-flux candidates
+- [#4243](https://github.com/oscar-system/Oscar.jl/pull/4243) Support computation of basis of H^(2,2) and H^4
+
+### Groups
+
+- [#4659](https://github.com/oscar-system/Oscar.jl/pull/4659) Implement `transitivity` and `rank_action` for $G$-sets
```

Comment:
```suggestion
```
This was merged after the release-1.3 branch was created and has no backport label